lighthouse-core/lib/styles-helpers.js

```diff
@@ -104,29 +104,25 @@ ${parsedContent.selector} {
 }
 
 /**
- * Returns a array of all CSS prefixes and the default CSS style names.
+ * Returns an array of all CSS prefixes and the default CSS style names.
  *
- * @param {string|Array<string>=} propNames CSS text content.
- * @param {Array<string>=} propsNamesWithPrefixes The parsed version content.
- * @return {{styleRule: string, location: string}} Formatted output.
+ * @param {string|Array<string>=} propNames CSS property names.
+ * @return {Array<string>=} CSS property names with and without vendor prefixes.
  */
-function addWebPrefixes(propsNames) {
+function addVendorPrefixes(propsNames) {
   const vendorPrefixes = ['-o-', '-ms-', '-moz-', '-webkit-'];
 
   let propsNamesWithPrefixes = [];
```

Comment:
if you init'd `let propsNamesWithPrefixes = propsNames;` here you could just `return propsNamesWithPrefixes;` at the bottom. Remove the extra concat.